BranchSDK/src/BranchIO/AppInfo.cpp

```diff
@@ -24,10 +25,13 @@ AppInfo::setAppVersion(const String& appVersion) {
     return doAddProperty(Defines::JSONKEY_APP_VERSION, appVersion.str());
 }
 
+#pragma warning(disable: 4995)
 AppInfo&
 AppInfo::setDeveloperIdentity(const String& identity) {
-    return doAddProperty(Defines::JSONKEY_APP_DEVELOPER_IDENTITY, identity.str());
```

Comment:
Since setDeveloperIdentity is deprecated and  JSONKEY_APP_DEVELOPER_IDENTITY is no longer used, should we just remove all code that uses JSONKEY_APP_DEVELOPER_IDENTITY and also remove JSONKEY_APP_DEVELOPER_IDENTITY from defines.h and defines.cpp ?

Reply:
`developer_identity` is a meaningful field to the API. It's just not used in the code. It may be present in the responses to some requests.